src/diffusers/pipelines/__init__.py

```diff
@@ -117,6 +117,7 @@
         StableDiffusionXLPipeline,
     )
     from .t2i_adapter import StableDiffusionAdapterPipeline
+    from .t2i_adapter import StableDiffusionAdapterPipeline
```

Comment:
Needs to be modified to the XL pipeline.

Reply:
@sayakpaul Do you know the reason for this? It seems that I haven't modified the related files.
![image](https://github.com/huggingface/diffusers/assets/54032224/7f2650d0-be8c-4d37-91cc-700b9b81f2d5)